e2e/test/scenarios/onboarding/setup/setup.cy.spec.js

```diff
@@ -151,6 +151,8 @@ describe("scenarios > setup", () => {
         .should("not.be.disabled")
         .click();
 
+      cy.findByText("I'll add my data later").click();
```

Comment:
you can add `// eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage` before this line